app/src/main/java/io/appium/settings/Settings.java

```diff
@@ -70,12 +79,104 @@ public void onCreate(Bundle savedInstanceState) {
             LocationTracker.getInstance().start(this);
         }
 
+        // REQUIRED STEPS to activate screenrecord feature:
+        // adb shell pm grant io.appium.settings android.permission.RECORD_AUDIO
+        // adb shell pm grant io.appium.settings android.permission.WRITE_EXTERNAL_STORAGE
+        // adb shell appops set io.appium.settings PROJECT_MEDIA allow
+
+        // TO START SCREENRECORD:
+        // adb shell am start -n "io.appium.settings/io.appium.settings.Settings" -a io.appium.settings.recording.ACTION_START --es "recording_filename" "abc.mp4"
+
+        // TO STOP SCREENRECORD:
+        // adb shell am start -n "io.appium.settings/io.appium.settings.Settings" -a io.appium.settings.recording.ACTION_STOP
+
+        // handle screen recording
+        Intent openingIntent = getIntent();
+        if (openingIntent != null && openingIntent.getAction() != null) {
```

Comment:
I'd rather move this block into a separate helper method

Reply:
done